### PR TITLE
Release CSV Updates for Inventory dashboard

### PR DESCRIPTION
**Pull request by ronniegeraghty**
Updates to the Release CSV for the Inventory Dashboard Project.

### CODE REVIEW

**Comment by weshaggard** [comment]
@ronniegeraghty the longer you keep these changes in another branch the more likely there will be more fun merge conflicts to resolve. I'd suggest trying to get the conflicts resolved again and let's get it in quickly.

**Comment by ronniegeraghty** [comment]
> @ronniegeraghty Ronnie Geraghty FTE the longer you keep these changes in another branch the more likely there will be more fun merge conflicts to resolve. I'd suggest trying to get the conflicts resolved again and let's get it in quickly.

Was just waiting for you to be back to approve the PR. And now that I'm back from vacation, I'll get the current conflicts resolved and then I'll instruct the team to go back to opening their own PRs into main.

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -1,12 +1,12 @@
 "Package","VersionGA","VersionPreview","DisplayName","ServiceName","RepoPath","MSDocs","GHDocs","Type","New","PlannedVersions","LatestGADate","FirstGADate","Support","EOLDate","Hide","Replace","ReplaceGuide","MSDocService","ServiceId","Notes"
-"Azure.AI.AnomalyDetector","","3.0.0-preview.6","Anomaly Detector","Anomaly Detector","anomalydetector","NA","","client","true","","","","","","","Microsoft.Azure.CognitiveServices.AnomalyDetector","","","",""
+"e","","3.0.0-preview.6","Anomaly Detector","Anomaly Detector","anomalydetector","NA","","client","true","","","","","","","Microsoft.Azure.CognitiveServices.AnomalyDetector","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:2]: Looks like a bad merge

File: _data/releases/latest/java-packages.csv [hunk 2]
```diff
@@ -1,19 +1,19 @@
 "Package","GroupId","VersionGA","VersionPreview","DisplayName","ServiceName","RepoPath","MSDocs","GHDocs","Type","New","PlannedVersions","LatestGADate","FirstGADate","Support","EOLDate","Hide","Replace","ReplaceGuide","MSDocService","ServiceId","Notes"
-"azure-ai-anomalydetector","com.azure","","3.0.0-beta.5","Anomaly Detector","Anomaly Detector","anomalydetector","","","client","true","","","","","","","","","","",""
+"r","com.azure","","3.0.0-beta.5","Anomaly Detector","Anomaly Detector","anomalydetector","","","client","true","","","","","","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:2]: Looks like another bad merge

Comment (ronniegeraghty) [reply _data/releases/latest/java-packages.csv]: Change this back and found a similar issue in the java csv, also related to Anomaly Detector and changed it.

**Pull request merged**